example/example-event/example-event-listener/src/main/java/uk/gov/justice/services/example/cakeshop/event/listener/CakeMadeEventListener.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.justice.services.example.cakeshop.event.listener;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import uk.gov.justice.services.core.annotation.Component;
+import uk.gov.justice.services.core.annotation.Handles;
+import uk.gov.justice.services.core.annotation.ServiceComponent;
+import uk.gov.justice.services.messaging.Envelope;
+
+@ServiceComponent(value = Component.EVENT_LISTENER)
+public class CakeMadeEventListener {
+
+    Logger logger = LoggerFactory.getLogger(CakeMadeEventListener.class);
+
+    @Handles("cakeshop.events.cake-made")
+    public void handle(final Envelope envelope) {
+
+        logger.info("=============> Inside cake-made Event Listener");
```

Comment:
We haven't really implemented make-cake yet. Only add-recipe. Handlers have been created just to ensure it doesn't crash.